README.md

```diff
@@ -1,6 +1,110 @@
+guide for the design and implementation of features during and after their
+development. A secondary audience is general users and developers who are
+interested in how and why a feature was designed and implemented.
+
+## RFC Workflow
+[rfc-workflow]: #rfc-workflow
+
+- **Community Discussion**: A need or issue is brought to the
+  [discussion forum](https://discuss.tvm.apache.org). During this phase, the
+  developer and user community can discuss the need for and requirements of the
+  RFC
+- **Pull Request**: After or concurrent with the conversation on the discussion
+  forum, a pull request is created using the format prescreibed by the
+  [RFC Template](https://github.com/apache/tvm-rfcs/blob/main/0000-template.md)
+    - Discussion about the details of the RFC can continue in the pull request.
+	- A committer of the corresponding area will approve and merge the RFC.
+      Normally the corresponding committer will become the shepherd of the
+      implementation PRs.
+	- RFCs are numbered consecutively based on their order of proposal,
+      regardless of if they are accepted or postponed.
+    - Legacy RFCs will be numbered consecutively, prefixed with the letter
+      `L` to indicate it is a legacy RFC. For example, `L0001`.
+	- A successful RFC will include an overview with the problem the RFC is
+      attempting to address, a proposed solution that describes the design and
+      implementation strategy, and a timeline for completion. Optional sections can
+      include (but are not limited to) alternatives that were considered, security
+      considerations, and open problems that the RFC does not solve.
+	- It is expected that RFCs will change, as part of the feedback process and
+      as new implementation details arise. Changes to the RFC should not be squashed
+      or force pushed in order to retain change and discussion history.
+    - The formal RFC may link back to the original discussion if there is
+      additional context or discussion, but all of the final feature design
+      must be completely described in the pull request.
+- **Tracking Issue**: Upon merging a RFC, a tracking issue will be created where
+  implementors can continue sharing implementation details (including links to
+  pull requests). The issue will be closed when the RFC is either completed or
+  abandoned.
+- **Implementation**: Work will begin on the RFC, with
+  pull requests linking back to the tracking issue. Upon completion of the RFC,
+  the tracking issue will be closed and the RFC will be moved to the
+  docs/rfc/completed directory.
+- **Changes**: It is not uncommon for design changes to be required during or
+  after the initial implementation. If this is the case, the RFC should be
+  updated to reflect the change. In the instance where the change is a
+  significant addition rather than a simple modification, a new RFC should be
+  posted.
+- **Postponement**: An RFC may be postponed either
```

Comment:
This seems to be a departure from the rust RFC model https://github.com/rust-lang/rfcs. I wonder if we want to make the process more consistent with the rust RFC process. In particular, a RFC pull request itself can be postponed, but the implementation of RFC should follow a separate process. See https://github.com/rust-lang/rfcs#implementing-an-rfc

Maintaining the state of the RFC implemention inside the rfc repo can also be a maintaince overhead. The status implementation should be tracked in a tracking issue.